### PR TITLE
Add support for databricks

### PR DESCRIPTION
**Pull request by rlh1994**

### CODE REVIEW

File: macros/split_events.sql [hunk 1]
```diff
@@ -98,7 +98,14 @@ where
     and {{ snowplow_utils.is_run_with_new_events("snowplow_web") }}
 {% endmacro %}
 
-{# {% macro databricks__split_events(event_name, flat_cols, sde_col, sde_keys, sde_types, context_cols, context_keys, context_types, context_aliases) %}
+{% macro databricks__split_events(event_name, flat_cols, sde_col, sde_keys, sde_types, context_cols, context_keys, context_types, context_aliases) %}
+
+{# Remove down to major version for Snowflake columns, drop 2 last _X values #}
```
Comment (emielver) [inline macros/split_events.sql:103]: I guess they're not Snowflake columns but the same principle applies here, might be worth updating the comment

File: macros/users_table.sql [hunk 2]
```diff
@@ -72,3 +72,52 @@ from
 where
     rn = 1
 {% endmacro %}
+
+{% macro databricks__users_table(user_cols, user_keys, user_types) %}
+
+{# Remove down to major version for Snowflake columns, drop 2 last _X values #}
```
Comment (emielver) [inline macros/users_table.sql:78]: Same here about Snowflake

File: macros/users_table.sql [hunk 3]
```diff
@@ -72,3 +72,52 @@ from
...
+{%- for ind1 in range(user_keys|length) -%}
+    {%- set user_key_clean = [] -%}
+    {%- for ind2 in range(user_keys[ind1]|length) -%}
+        {% do user_key_clean.append(re.sub(camel_string, '_', user_keys[ind1][ind2]).lower()) -%} 
+    {%- endfor -%}
+    {% do user_keys_clean.append(user_key_clean) -%} 
+{%- endfor -%}
+
+
+with users_ordered as (
```
Comment (emielver) [inline macros/users_table.sql:97]: Are they actually ordered? I don't see an ORDER BY statement or is that not necessarily what you mean by ordered?

File: macros/users_table.sql [hunk 4]
```diff
@@ -72,3 +72,52 @@ from
...
+    user_id is not null
+    and {{ snowplow_utils.is_run_with_new_events("snowplow_web") }}
+)
+
+select
+    * except (rn)
+from
+    users_ordered
+where
+    rn = 1
```
Comment (emielver) [inline macros/users_table.sql:122]: Not sure if worth adding but might be good to have a comment either here or in the readme explaining what we're doing, i.e. taking the latest user_id properties from the `base_events_this_run` table ordered by the `collector_tstamp`.

**Review by emielver — COMMENTED** [review]
Only small nitpicky comments, great work on this!

File: macros/users_table.sql (hunk 3 above)
Comment (rlh1994) [reply macros/users_table.sql:97]: I meant "ordered" in that there is a key that you can now order on, think it should have a different name?

File: macros/users_table.sql (hunk 4 above)
Comment (rlh1994) [reply macros/users_table.sql:125]: Added more info to macro description and the main docs

**Pull request merged**